x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/models/list/pipeline_element.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class PipelineElement {
```

Comment:
I found the name `PipelineElement` to be confusing. Does it cause name conflict problems to call it `Element` (much like the base class for the `*Statement` classes is `Statement`).

Reply:
> Does it cause name conflict problems

This was my motivation when I named it. I think we'll be ok to call it `Element` though.